pages-e2e/fixtures/pages12.3.0/next.config.js

```diff
@@ -5,4 +5,7 @@ module.exports = {
 		runtime: 'experimental-edge',
 	},
 	swcMinify: false,
+	env: {
+		frameworkVersion: '12',
```

Comment:
Should [pages13.0.0/next.config.js](https://github.com/cloudflare/next-on-pages/blob/9dbefa1b9aaf0b86a11d3a166ae7868f35dee727/pages-e2e/fixtures/pages13.0.0/next.config.js) have an equivalent framework version?

Reply:
this is sort of a hack, just because Next 12 doesn't seem to support some things (and/or we fail to support them, but it's not likely worth it to spend too much time trying to... I think?)

so I use this to change outputs slightly in the features so that I know when not to test stuff, truthfully more than a version value this is a flag, like `isNext12: boolean` (but I kept it generic as that might come in handy? 🤔)

adding it to `pages13.0.0` wouldn't change anything (since `process.env.frameworkVersion === '12'` would work identically) but I wouldn't mind adding it to all our fixtures just to make keep things consistent 🙂

Reply:
I've added the `frameworkVersion` to all the fixture's config files 🙂

This doesn't include the ones we generate on the fly, for those I'd need to implement codemod logic to update the generated configs, it's not too too complex but not too trivial either so I figured I could skip that for now (since it wouldn't even have any effect) and do that if/when actually needed, hopefully that's ok with you 🙂